src/content/en/2020/accessibility.md

```diff
@@ -226,227 +346,247 @@ Some assistive technologies for the Web include:
+It is fairly likely that a native `<button>` element would be a better choice, per the first rule of ARIA. It is also possible that the role has been added but the expected keyboard support has not been supplied, which would break the third rule of ARIA and would violate [WCAG 2.1.1, Keyboard](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html).
+
+{{ figure_markup(
+  caption='Mobiles sites assigning `role="button"` to `<div>` or a `<span>`',
+  content="8.28%",
+  classes="big-number",
+  sheets_gid="1693851651",
+  sql_file="div_span_with_button_or_link_role.sql"
+) }}
+
+We found that 8.27% of desktop pages and 8.28% of mobile pages had at least one occurrence of a `<div>` or a `<span>` element with `role="button"` explicitly defined. This act of adding ARIA roles, or a "[role-up](https://adrianroselli.com/2020/02/role-up.html)", is less ideal than using the correct native HTML element.
 
-We found that 15.50% of desktop pages and 14.62% of mobile pages contained at least one anchor element with `role="button"`. If a role has been applied to an element that should have its implicit role respected, such as giving a `role="button"` to a link (which has an implicit `role="link"`), this would break the 2nd rule of ARIA and would violate [WCAG 2.1.1, Keyboard](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html) if the correct keyboard behavior has not been implemented (links are not activated with the space key, where as buttons are).
+We found that 15.50% of desktop pages and 14.62% of mobile pages contained at least one anchor element with `role="button"`. If a role has been applied to an element that should have its implicit role respected, such as giving a `role="button"` to a link (which has an implicit `role="link"`), this would break the second rule of ARIA. It would also violate [WCAG 2.1.1, Keyboard](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html) if the correct keyboard behavior has not been implemented (links are not activated with the space key, whereas buttons are).
 
 In the vast majority of these cases, a better pattern than explicitly defining `role="button"` on the element in question would be to leverage the native HTML `<button>` element.
 
-##### Navigation
+{# TODO (Authors) Repetition of above "It is fairly likely that a native `<button>` element would be a better choice, per the first rule of ARIA.". Should we remove one instance or happy to keep both? #}
```

Comment:
@oluoluoxenfree / @alextait1

Reply:
I see how it's a bit repetitive. But the first instance it is referring to the first rule of ARIA (use the native thing) and the second instance is more, given all the reasons not just the first rule of ARIA, use a button.

Reply:
What about acknowledging the repetition with “again”, or “similar to previous”?

“Again, in the vast majority of cases...”

Reply:
Good suggestion, updated